Makefile

```diff
@@ -12,7 +12,7 @@ export REPORT_DIR?=$(CURDIR)/reports/$(shell date +%Y-%m-%d-%H-%M-%S)
 export KUBECTL=$(shell command -v kubectl)
 
 CADVISOR_NAMESPACE := cadvisor
-LOKI_NAMESPACE := observatorium-logs-test
+OBSERVATORIUM_NAMESPACE := observatorium-logs-test
```

Comment:
This benchmark suite is dedicated to `loki` and not `observatorium`. Let's not rewrite anything to observatorium.